src/components/EditButtonsArea/EditButtonsArea.js

```diff
@@ -1,36 +1,63 @@
 import "./EditButtonsArea.css";
 import { useFolderNameContext } from "../../context/FolderNameContext";
+import { useState } from "react";
+import ModalEditFolderName from "../Modal/ModalEditFolderName";
+import ModalDeleteFolder from "../Modal/ModalDeleteFolder";
+import ModalShareFolder from "../Modal/ModalShareFolder";
 
 const EditButtonsArea = () => {
   const { folderName } = useFolderNameContext();
+  const [showModalEditFolderName, setShowModalEditFolderName] = useState(false);
+  const [showModalDeleteFolder, setShowModalDeleteFolder] = useState(false);
+  const [showModalShareFolder, setShowModalShareFolder] = useState(false);
```

Comment:
[중요도: 중간] boolean의 경우는 앞에 is를 붙여 isShowModalEditFolderName 형식으로 이름을 바꾸는 것도 좋아보입니다. (boolean의 경우는 보통 is를 붙이는게 개발자 사이의 국룰로도 취급할 때가 있더라구요. 백엔드는 그게 고정되는 경우도 있습니다 ㅎㅎ)

```suggestion
 // string 형태로 들어가 'editFolderName', 'deleteFolder', 'shareFolder' 혹은 undefined가 들어가게 한다.
  const [showModalType, setShowModalType] = useState();
  .
  .
  .
  {showModalType === 'editFolderName' && (
      <ModalEditFolderName handleClose={handleModalEditFolderName} />
    )}
    {showModalType === 'deleteFolder' && (
      <ModalDeleteFolder handleClose={handleModalDeleteFolder} />
    )}
    {showModalType === 'shareFolder' && (
      <ModalShareFolder handleClose={handleModalShareFolder} />
    )}
```
[중요도: 중간] 위의 코드처럼 변환해보는 것도 나쁘지 않습니다. 어디까지나 제안이니 참고해보세요~
장점: 하나의 state로 modal 관리들이 가능하다.
단점: 2개 이상의 modal이 한 화면에 노출되어야 한다면 문제가 생긴다.